lib/chef/mixin/powershell_out.rb

```diff
@@ -34,7 +34,14 @@ def powershell_out(*command_args)
         script = command_args.first
         options = command_args.last.is_a?(Hash) ? command_args.last : nil
 
-        run_command_with_os_architecture(script, options)
+        result = run_command_with_os_architecture(script, options)
+
+        if result.stderr.include? "A positional parameter cannot be found that accepts argument"
+          raise "Please use single escape for special characters in powershell_out."
```

Comment:
We should raise ArgumentError or something more specific from `lib/chef/exceptions.rb`. Is there some argument we could include in this message to be more specific about where the error was. Without looking, maybe `script` or `options`?